cnc_ctrl_v1/Motor.cpp

```diff
@@ -127,24 +122,33 @@ void Motor::directWrite(int voltage){
     Write directly to the motor, ignoring if the axis is attached or any applied calibration.
     */
     
-    if (voltage > 0){
-        digitalWrite(_pin1 , HIGH);
-        digitalWrite(_pin2 , LOW );
-    }
-    else if (voltage == 0){
-        voltage = voltage;
-    }
-    else{
-        digitalWrite(_pin1 , LOW);
-        digitalWrite(_pin2 , HIGH );
-    }
-    
     if(_pwmPin == 12){
-        voltage = abs(voltage);
-        voltage = map(voltage, 0, 255, 0, 1023);  //Scales 0-255 to 0-1023
-        Timer1.pwm(2, voltage);  //Special case for pin 12 due to timer blocking analogWrite()
+        if (voltage > 0){
+            analogWrite(_pin1 , abs(voltage));
```

Comment:
Is it safe to do PWM on these pins? I makes sense that it would be, but all the examples for using the part did the PWM on the other pin